src/hlsBinaries.ts

```diff
@@ -380,21 +390,31 @@ async function getLatestHLS(
  * @param logger Logger for feedback.
  * @returns The GHC version, or fail with an `Error`.
  */
-export async function getProjectGHCVersion(wrapper: string, workingDir: string, logger: Logger): Promise<string> {
+export async function getProjectGHCVersion(
+  toolchainBindir: string,
+  workingDir: string,
+  logger: Logger
+): Promise<string> {
   const title = 'Working out the project GHC version. This might take a while...';
   logger.info(title);
+
   const args = ['--project-ghc-version'];
 
+  const newPath = await addPathToProcessPath(toolchainBindir, logger);
+  const environmentNew: IEnvVars = {
+    PATH: newPath,
+  };
+
   return callAsync(
-    wrapper,
+    'haskell-language-server-wrapper',
```

Comment:
It is guaranteed that `haskell-language-server-wrapper` is on the PATH at that moment, we should document that somehow, imo

Reply:
This codepath doesn't happen if the users sets`manageHLS` to `PATH`. They will get this error instead:
![Screenshot_2022-03-20_15-48-08](https://user-images.githubusercontent.com/1241845/159168198-2364de2b-461c-4a8e-92bc-e8edf54e25cd.png)

If `manageHLS` is set to `GHCup`, then yes, this is guaranteed.